source/backend/api/Services/AcquisitionFileService.cs

```diff
@@ -87,6 +87,44 @@ public Paged<PimsAcquisitionFile> GetPage(AcquisitionFilter filter)
             return _acqFileRepository.GetPage(filter, userRegions, personId);
         }
 
+        public List<AcquisitionFileExportModel> GetAcquisitionFileExport(AcquisitionFilter filter)
+        {
+            _logger.LogInformation("Searching for acquisition files...");
+            _logger.LogDebug("Acquisition file search with filter", filter);
```

Comment:
nit: I think you need to include a placeholder in the log statement - i.e.

```csharp
_logger.LogDebug("Acquisition file search with filter: {filter}", filter);
```

Reply:
Agreements doesn't has one...

![image](https://github.com/bcgov/PSP/assets/4305638/8052b352-ab10-4833-b4de-aab3a277a689)

Now that I think about it, shouldn't it be ??

`_logger.LogDebug("Acquisition file search with filter: {0}", filter);  `

Reply:
updated